helm_deploy/values-preprod.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-# Per environment values which override defaults in hmpps-template-kotlin/values.yaml
+# Per environment values which override defaults in hmpps-manage-a-warrant-folder-api/values.yaml
 
 generic-service:
   ingress:
-    host: hmpps-template-kotlin-preprod.hmpps.service.justice.gov.uk
+    host: hmpps-manage-a-warrant-folder-api-preprod.hmpps.service.justice.gov.uk
```

Comment:
You don't necessarily have to start with `hmpps-` in the url as there is a .hmmps. in the url path.